### ISSUE TITLE
promotion in Dataverse guides, repo under IQSS

### ISSUE THREAD

**Issue by pdurbin** [gubi/Dataverse-PHP-API#2]
Hi! Thanks for making a PHP wrapper for the Dataverse API! I noticed it in Dataverse Slack and in a [tweet][] and I'm wondering if you'd like:

- promotion of this library in a future version of http://guides.dataverse.org/en/4.9.4/api/client-libraries.html and
- a repo under @IQSS called "dataverse-client-php" to match the other community-contributed libraries for Python, R, and Java (listed on that "Client Libraries" page)

Also, from a quick search on https://packagist.org I found https://github.com/nacasha/dataverse-api by @nacasha so maybe you two could consider coordinating.

Thanks again!

[tweet]: https://twitter.com/call_user_func/status/1064138688068636672

**Comment by gubi** [comment]
Hi and thank you for your contact!
Yes, for now is a draft version I use in combination with this package: https://github.com/bioversity/Excel-2-Dataverse.
Currently it has some bugs and not tested features yet, but once completed I would have looked for you... :)

I'm very interested to give you this package and  - if you want - feel free to consider me part of the community :)
Lastly, as you suggested I'll contact @nacasha in order to work together on a single package.

Is there a way to test API commands on a test environment?
I need an endpoint for sending get/post/put/delete commands without damages. For now I'm using https://apitest.dataverse.org, but the interface doesn't works well and I'm going a bit in blind.

Let me know
Very happy for your "issue"

**Comment by pdurbin** [comment]
@gubi yes! You should used https://demo.dataverse.org instead. This mentioned in passing at http://guides.dataverse.org/en/4.9.4/api/intro.html#testing

No rush on anything. Thanks!!
